recipes/rubicon-ml/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "rubicon-ml" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rubicon-ml-{{ version }}.tar.gz
+  sha256: 8c2c4e9247ae54768f53cfd89e3e352f02b78e94cc03b350171d66e9804ac875
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - rubicon=rubicon.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - click >=7.1
+    - dask[dataframe] >=2.12.0
```

```suggestion
    - dask >=2.12.0
    - pandas >=1.0
```

you probably want this

---

thanks, updated